app/javascript/components/molecules/Popover.js

```diff
@@ -47,20 +46,32 @@ const Popover = ({ buttonElement, children }) => {
 
   return (
     <>
-      {buttonElement({ ref: buttonRef, onClick: () => setShowPopper(!showPopper) })}
+      {buttonElement({
+        ref: setReferenceElement,
+        onClick: () => {
+          setShowPopper(!showPopper)
+          onButtonClick && onButtonClick()
+        }
+      })}
       {showPopper && (
         <div
           className='popper-container'
-          ref={popperRef}
+          ref={setPopperElement}
           style={styles.popper}
           {...attributes.popper}
         >
-          <div ref={setArrowRef} id='arrow' />
+          <div ref={setArrowElement} style={styles.arrow} id='arrow' />
           {children}
```

Comment:
Do you know what this looks like when children is null? If it shouldn't be null can you mark children as required in the props?

Reply:
it works it just looks empty 
![image](https://user-images.githubusercontent.com/11825994/96631586-80f02f80-12e4-11eb-9c99-2f2957ffa77f.png)